internal/hierarchyconfig/validator.go

```diff
@@ -129,6 +132,38 @@ func (v *Validator) handle(ctx context.Context, log logr.Logger, req *request) a
 		return webhooks.Deny(metav1.StatusReasonForbidden, reason)
 	}
 
+	allErrs := field.ErrorList{}
+
+	fldPath := field.NewPath("spec", "labels")
+	for _, l := range req.hc.Spec.Labels {
+		if err := validateMetaKey(l.Key, fldPath); err != nil {
```

Comment:
Instead of `err`, maybe use `fe` for `field.Error` since it's not a standard Go `error`? Not a huge deal, but whenever I see `err` I just assume it's `error`.

Reply:
I went for `fldErr`, and hope that is ok. Two letter variable names can be a bit "tricky". PTAL!

Reply:
np!